### DISCUSSION TITLE
Errors in running the risingbubble2D tutorial case in GeoChemFoam-5.0

### DISCUSSION THREAD

**Discussion by lukc0331** [GeoChemFoam/GeoChemFoam#10] [Q&A] [answered]
Dear Dr. Hannah,

I am following the user guide and trying to run the tutorial case "risingbubble2D", but encountered some errors.

First, it seems that the interTransferFoam solver cannot be compiled successfully, and the error message is as follows, which says that some files cannot be found.
<img width="721" alt="微信图片_20230727001924" src="https://github.com/GeoChemFoam/GeoChemFoam/assets/140630196/87dd5938-eacf-4010-a613-ab0ede2ad87e">

Secondly, there are two cases (Case0 and CaseTransfer) under the risingbubble2D folder. It could successfully run for Case0 after executing createMesh.sh, initCase0.sh and finally runCase0.sh (the solver is interGCFoam). However, when it comes to CaseTransfer, initCaseTransfer.sh cannot be executed successfully. The error message says three problems exist: unknown command bc, no Species file under 0 folder, and alpha.phase1 cannot be read. 

<img width="515" alt="1690386146059" src="https://github.com/GeoChemFoam/GeoChemFoam/assets/140630196/b7fba160-74ce-44c9-bea4-a0307a9531f9">

I would really appreciate it if you could give me some help.

Thanks.

**Comment by julienmaes** [comment]
Looks like you have not installed the libraries. Did you run the Allwmake from the GeoChemFoam directory?

**Reply by lukc0331** [reply]
Yes, I have run Allwmake from the GeoChemFoam directory. I re-checked it and found that the previous installation failed because the RM_interface_F.cpp file went wrong. According to the error message, the problem lay in line 253 that l1 cannot be directly compared with zero and there should be (unsigned int) * before l1. Then I recompiled and the libraries were successfully installed.

However, when I try to run the tutorial case risingbubble2D, the same errors occurred when I ran ./initCaseTransfer.sh.
<img width="681" alt="1690424044096" src="https://github.com/GeoChemFoam/GeoChemFoam/assets/140630196/3a9670a6-3b8a-4505-ae6b-382efca16a14">

**Reply by lukc0331** [reply, marked as answer]
Get it, I need to install command bc, which is used for basic calculation.
